examples/e2e/pipelines/batch_inference.py

```diff
@@ -59,16 +59,16 @@ def e2e_use_case_batch_inference():
     df_inference = inference_data_preprocessor(
         dataset_inf=df_inference,
         preprocess_pipeline=ExternalArtifact(
+            name="preprocess_pipeline",
             pipeline_name=MetaConfig.pipeline_name_training,
-            artifact_name="preprocess_pipeline",
         ),
         target=target,
     )
     ########## DataQuality stage  ##########
     report, _ = evidently_report_step(
         reference_dataset=ExternalArtifact(
+            name="dataset_trn",
             pipeline_name=MetaConfig.pipeline_name_training,
-            artifact_name="dataset_trn",
```

Comment:
These changes are not effective, cause E2E template is auto-updated once tests pass and to make them pass we need a fix on template side + new release tag here.
That's why test is failing.

Reply:
Ok then I undo the changes in `examples/e2e` and change them in the e2e repo instead?

Reply:
Yes, pelase